ambersim/utils/io_utils.py

```diff
@@ -69,7 +75,7 @@ def load_mj_model_from_file(
             output_name = "/".join(str(filepath).split("/")[:-1]) + "/_temp_xml_model.xml"
             save_model_xml(filepath, output_name=output_name)
             mj_model = _modify_robot_float_base(output_name)
-            Path.unlink(output_name)
+            Path(output_name).unlink()
```

Comment:
Example: this is actually just a straight up error that I resolve in a sibling branch, but I have no idea how it passed CI for so long. It actually occurs in a few other places, too.